src/pgen/cluster/agn_triggering.cpp

```diff
@@ -153,25 +166,31 @@ void AGNTriggering::ReduceColdMass(parthenon::Real &cold_mass,
 
   const bool remove_accreted_mass = remove_accreted_mass_;
 
-  Real md_cold_mass = 0;
+  
 
   parthenon::par_reduce(
       parthenon::loop_pattern_mdrange_tag, "AGNTriggering::ReduceColdGas",
       parthenon::DevExecSpace(), 0, cons_pack.GetDim(5) - 1, kb.s, kb.e, jb.s, jb.e, ib.s,
       ib.e,
       KOKKOS_LAMBDA(const int &b, const int &k, const int &j, const int &i,
-                    Real &team_cold_mass) {
+                    Real &team_cold_mass, Real &team_total_mass) {
```

Comment:
Looks like the abstraction for a par reduce to two scalars is currently not available within Parthenon (though we're working on this).
In the meantime I suggest to use a raw Kokkos pattern, i.e., 
```c++
    Kokkos::parallel_reduce(
        "AGNTriggering::ReduceColdGas",
        Kokkos::MDRangePolicy<Kokkos::Rank<4>>(
            DevExecSpace(), {0, kb.s, jb.s, ib.s},
            {cons_pack.GetDim(5), kb.e + 1, jb.e + 1, ib.e + 1},
            {1, 1, 1, ib.e + 1 - ib.s}),
        KOKKOS_LAMBDA(const int &b, const int &k, const int &j, const int &i,
                      Real &team_cold_mass, Real &team_total_mass) {
```